tests/lib/utils/utils-test.js

```diff
@@ -126,6 +126,14 @@ describe('isConditionalExpression', () => {
   });
 });
 
+describe('isTaggedTemplateExpression', () => {
+  const node = parse('test = hbs`lorem ipsum`;').right;
+
+  it('should check if node is a tagged template expression', () => {
+    assert.ok(utils.isTaggedTemplateExpression(node));
+  });
```

Comment:
What about an additional test case - where node is *not* a tagged template expression?